son/content/intermediate_outcomes/routes_into_work_(16_to_29_years)/entry_to_higher_education/3.0.md

```diff
@@ -0,0 +1,158 @@
+
+        ::: visualisation vis_type="chart" section_name="By year"
+            {
+                "code": "IN35",
+                "title": "Percentage of 18 to 20 year olds enrolled in higher education, by socio-economic background (UK, 2014 to 2024, 3-year rolling averages)",
+                "tooltipTitle": "",
+                "data": "3.0/IN22-3.0-entry-to-higher-education--by-moving-average--chart-format.csv",
+                "height": 450,
+                "type": "liney",
+                "xkey": "average_window",
+                "ykey": "value",
+                "zkey": "secondary_split_value",
+                "sort": "time_period",
+                "scale": "%",
+                "rounding": "0dp",
+                "lowerConfidence": "lci",
+                "upperConfidence": "uci",
+                "xgrid": false,
+                "ygrid": true,
+                "xticksSmallScreen": "2nd",
+                "legend": true,
+                "colourScheme": ["#1a9e1c", "#00703c", "#eb652e", "#0096ff", "#0f265c", "#808080"],
+                "range": [0, 60],
+                "margin": [10, 20, 5, 5],
+                "maxLabelLength": 55,
+                "rolloverBehaviour": "fade",
+                "clickBehaviour": "isolate"
+            }
+        :::
+    :::
+
+    ::: tab section_name="Moving average by socio-economic background" tab_name="Data"
+        ::: data_table
+            {
+                "title": "Percentage of 18 to 20 year olds enrolled in higher education, by socio-economic background (UK, 2014 to 2024, 3-year rolling averages)",
+                "dataTable": "3.0/IN22-3.0-entry-to-higher-education--by-moving-average--table-format.csv",
+                "dataTableAlignColumns": ["left", "right", "right", "right", "right", "right", "right"],
+                "dataTableDecimalPlaces": [null, 1, 1, 1, 1, 1, 1]
+            }
+        :::
+    :::
+
+    ::: tab section_name="Moving average by socio-economic background" tab_name="Download"
+        ::: download_section :::
+    :::
+:::
+
+
+## About the data
+
+### Data source
+Office for National Statistics, Labour Force Survey (LFS) 2022
+
+### Time period
+* 2024 (By socio-economic background)
+* 2014 and 2022 (By socio-economic background – 3-year rolling averages)
+
+### Geographic area
+UK
+
+### What the data measures
+The data shows the percentage of 18 to 20 year olds in the UK who began studying in higher education (HE) from 2014 to 2024. HE includes degree level qualifications and foundation degrees. 
+
+### Things you need to know
+The data is weighted using the LFS person weights. The error bars show 95% [confidence intervals](/about-our-analysis#confidence-intervals).
+
+The ratios are the percentage of 18 to 20 year olds from higher professional backgrounds in HE, divided by the percentage from lower working backgrounds.
+
+### Type of data
+Survey data

```

Comment:
With all this stuff at the bottom, feel free to update it if you know what needs to change.
Or feel free to leave it, since Spence and Alex (guided by the SMC team) will update this.
I don't mind either way - but don't spend too long figuring out what it should be, because someone else will review it anyway